### DISCUSSION TITLE
Lab 3 Feedback

### DISCUSSION THREAD

**Discussion by ering0427** [ering0427/Interactive-Lab-Hub#17] [General]
Thanks for any feedback!

**Comment by HeidiHe** [comment]
Speak communication is very helpful during exercises because it frees one's hands and also prevents one's sweaty body from contacting the device. It is very smart to put the speak-to-text device in the context of working out. My question is where do you consider the device to be? Is it a wearable device or something attached to the workout pad? Would you also consider displaying the demo of the moves on it as well?

**Comment by YuanhaoZhu** [comment]
The idea of using the accelerometer to detect if the user is holding the gesture still is a good feature that most of the current work-out digital products don't have. I think a good stretch for this project is to connect the Bluetooth headphone since the wired headphone can be tangled while you work out.
